apps/src/lib/ui/accounts/ChangeEmailController.js

```diff
@@ -114,8 +115,9 @@ export default class ChangeEmailController {
       };
       this.form.on('ajax:success', onSuccess);
       this.form.on('ajax:error', onFailure);
-      this.form.find('#change-email-modal_user_email').val(userAge < 13 ? '' : newEmail);
+      this.form.find('#change-email-modal_user_email').val(this.userAge < 13 ? '' : newEmail);
```

Comment:
Just checking that this age stays at 13, and doesn't need to change to 16, as we did with users signing up for petition?

Reply:
That's a good question.  So far I've been preserving existing behavior, but I'm not sure why we do this at all - for students 13 or older we include the cleartext email in-flight and then immediately discard it on the server.  Seems safer to just never include the cleartext email for students.  Thoughts?

Reply:
I think less sent the better, but I'm not very familiar with this flow.  @poorvasingal is my go-to on email policy :)